tests/test_zarr_encryption_ipfs.py

```
@@ -7,26 +7,38 @@
 import pandas as pd
 import xarray as xr
 import pytest
+import zarr
+import json
 from numcodecs import register_codec
+# from zarr.codecs import BytesCodec
+from numcodecs.zarr3 import BytesBytesCodec, Blosc
```

Ruff (0.8.2)

`numcodecs.zarr3.BytesBytesCodec` imported but unused

Remove unused import: `numcodecs.zarr3.BytesBytesCodec`

(F401)